crates/workspace/src/item.rs

```diff
@@ -36,7 +36,7 @@ use ui::Element as _;
 pub const LEADER_UPDATE_THROTTLE: Duration = Duration::from_millis(200);
 
 #[derive(Deserialize)]
-pub struct ItemSettings {
```

Comment:
Let's revert this name change. We refer to these as 'Item's throughout the rest of the codebase, and I think consistency is important.

Reply:
No problem, but the idea was that this struct is now fully dedicated to tabs settings, it's even exposed to users as `tabs`. It does not (and I guess it will not) have any non-tab-related settings. If there'll be some other item-, but not tab- related settings, they'll probably go to another struct.

Reply:
Reverted